src/helpers/unicode.js

```diff
@@ -40,18 +37,89 @@ export const KEY_CODES = {
   F10: 121,
   F11: 122,
   F12: 123,
-  A: 65,
-  C: 67,
-  D: 68,
-  F: 70,
+  F13: 124,
+  F14: 125,
+  F15: 126,
+  F16: 127,
+  F17: 128,
+  F18: 129,
+  F19: 130,
+  HOME: 36,
+  INSERT: 45,
   L: 76,
+  MEDIA_NEXT: 176,
+  MEDIA_PLAY_PAUSE: 179,
+  MEDIA_PREV: 177,
+  MEDIA_STOP: 178,
+  NULL: 0,
+  NUM_LOCK: 144,
   O: 79,
   P: 80,
+  PAGE_DOWN: 34,
+  PAGE_UP: 33,
+  PAUSE: 19,
+  PERIOD: 190,
   S: 83,
+  SCROLL_LOCK: 145,
+  SHIFT: 16,
+  SPACE: 32,
+  TAB: 9,
   V: 86,
   X: 88,
+  Y: 89,
+  Z: 90,
```

Comment:
Generally, it's a great idea to sort the props alphabetically. But I feel that in that case, it is harder to read. Especially with the single letters in between :D WDYT about grouping those letters below. Something like:
```js
  SHIFT: 16,
  SPACE: 32,
  TAB: 9,
  A: 65,
  C: 67,
  D: 68,
  F: 70,
  L: 76,
  O: 79,
  P: 80,
  S: 83,
  V: 86,
  X: 88,
  Y: 89,
  Z: 90,
```

Reply:
Yes, I agree. It is something that might improve readability.
Alternatively, I've been thinking about sorting keys by keyCode.
Thanks, @budnix 💪!